accesshandler/pkg/providers/flask/flask.go

```diff
@@ -0,0 +1,168 @@
+package flask
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/identitystore"
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/common-fate/granted-approvals/accesshandler/pkg/types"
+	"github.com/common-fate/granted-approvals/pkg/gconfig"
+	"github.com/invopop/jsonschema"
+	"go.uber.org/zap"
+)
+
+type Provider struct {
+	providerType  gconfig.StringValue
+	ecsClient     *ecs.Client
+	ssoClient     *ssoadmin.Client
+	iamClient     *iam.Client
+	idStoreClient *identitystore.Client
+	orgClient     *organizations.Client
+	awsAccountID  string
+
+	// configured by gconfig
+	ecsClusterARN gconfig.StringValue
+
+	// sso instance
+	instanceARN gconfig.StringValue
+	// The globally unique identifier for the identity store, such as d-1234567890.
+	identityStoreID gconfig.StringValue
+	// The aws region where the identity store runs
+	ssoRegion gconfig.StringValue
+	ecsRegion gconfig.StringValue
+
+	// a role which can be assumed and has required sso permissions
+	ssoRoleARN       gconfig.StringValue
+	ecsAccessRoleARN gconfig.StringValue
+
+	options map[string][]types.Option
+}
+
+func (p *Provider) Config() gconfig.Config {
+	return gconfig.Config{
+
+		gconfig.StringField("type", &p.providerType, "The type of the provider to display in the UI"),
```

Comment:
Do we need this, was in @chrnorm initial prototype

Reply:
no we don't - this was only used for demo providers